Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteLayerTreeContext.mm

```diff
@@ -203,7 +203,14 @@
     if (paintedAnyBackingStore)
         m_nextRenderingUpdateRequiresSynchronousImageDecoding = false;
 
-    transaction.setCreatedLayers(moveToVector(std::exchange(m_createdLayers, { }).values()));
+    transaction.setCreatedLayers(WTF::map(std::exchange(m_createdLayers, { }).values(), [&](auto&& layer) {
+#if ENABLE(VIDEO_PRESENTATION_MODE)
+        if (layer.videoElementData.has_value())
```

Comment:
`.has_value()` isn't necessary here, since `std::optional<T>::operator bool` will check for that.